website/content/en/preview/concepts/_index.md

```diff
@@ -72,7 +72,7 @@ For more details on how Karpenter deletes nodes, see [Deprovisioning nodes](../t
 ### Upgrading nodes
 
 A straight-forward way to upgrade nodes is to set `ttlSecondsUntilExpired`.
-Nodes will be terminated after a set period of time and will be replaced with newer nodes.
+Nodes will be terminated after a set period of time and will be replaced with newer nodes with the most recent AMI if using AL2 or Bottlerocket.
```

Comment:
can we say: 

"...newer nodes using the latest [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) or the AMI specified in the `$LATEST` version of your launch template."